frontend/packages/client/src/components/Community/SubscribeCommunityButton.js

```diff
@@ -0,0 +1,75 @@
+import { useModalContext } from 'contexts/NotificationModal';
+import { useNotificationServiceContext } from 'contexts/NotificationService';
+import { useWebContext } from 'contexts/Web3';
+import { Svg } from '@cast/shared-components';
+import { ErrorModal, NotificationsModal, WalletConnect } from 'components';
+import classnames from 'classnames';
+
+export default function SubscribeCommunityButton({
+  communityId,
+  className,
+  size = 'small',
+}) {
+  const { openModal, closeModal } = useModalContext();
+  const { notificationSettings } = useNotificationServiceContext();
+  const isSubscribed = notificationSettings?.communitySubscription.some(
+    (c) => c === communityId
+  );
+
+  const { user } = useWebContext();
+
+  const onOpenModal = () => {
+    if (!user?.addr) {
+      openModal(
+        <ErrorModal
+          message="In order to subscribe to a community, you must first connect your Flow wallet."
+          title="Connect Wallet"
+          footerComponent={
+            <WalletConnect
+              closeModal={() => {
+                closeModal();
+              }}
+              expandToContainer
+            />
+          }
+          onClose={closeModal}
+        />,
+        { isErrorModal: true }
+      );
+    } else {
+      openModal(
+        <NotificationsModal communityId={communityId} onClose={closeModal} />,
+        {
+          classNameModalContent: 'rounded modal-content-sm',
+          showCloseButton: false,
+        }
+      );
+    }
+  };
+
+  const buttonIcon = isSubscribed ? 'BellFilled' : 'BellHidden';
+  const buttonClasses = classnames(
+    'button subscribe-community-button p-0 is-fullwidth full-height rounded-full',
+    {
+      'px-4': size === 'large',
+    }
+  );
+
+  const sizes = {
+    smaller: { height: 32, maxWidth: 32 },
+    small: { width: 40, height: 40 },
+  };
+
+  const containerStyles = sizes[size] ?? { height: 48, maxWidth: 125 };
+
+  return (
+    <div
+      className={`column p-0 is-narrow-tablet is-full-mobile ${className}`}
+      style={containerStyles}
+    >
+      <button className={buttonClasses} onClick={onOpenModal}>
+        <Svg name={buttonIcon} />
+      </button>
```

Comment:
on mobile the icon is breaking: 
<img width="343" alt="Screen Shot 2022-10-18 at 12 21 49" src="https://user-images.githubusercontent.com/7526740/196473470-23bc8f62-9f6b-4692-807d-0bc09f1b363c.png">

Reply:
this should fix it https://github.com/DapperCollectives/CAST/pull/650/commits/ef0ce77cf1cf1b7775df604ac30ffb1f73055665